packages/engine/src/worker/runner/python/package.py

```diff
@@ -1,28 +1,54 @@
+import importlib
 from pathlib import Path
+import sys
+import traceback
 
+import logging
 
 def get_pkg_path(pkg_name, pkg_type):
-    return Path("../../../simulation/package/{}/packages/{}/package.py".format(
+    # The engine should be started from the engine's root directory in the repo.
+    return Path("./src/simulation/package/{}/packages/{}/package.py".format(
```

Comment:
If this works, it works. But it's something I'd like to revisit at some point, not happy about all of our relative paths and such, especially since things like running Python with `-m` can affect it

Reply:
Yeah, this does rely on running the engine from a specific directory.